grails-app/services/gocharges/PaymentService.groovy

```diff
@@ -44,12 +45,13 @@ class PaymentService {
         payment.save(failOnError: true)
     }
 
-    public void confirm(Long id) {
+    public void confirmReceivedInCash(Long id) {
         Payment payment = PaymentRepository.query([id: id]).get()
 
         if (!payment) throw new BusinessException("Cobrança não encontrada")
 
         payment.status = PaymentStatus.RECEIVED
+        payment.billingType = PaymentBillingType.CASH
```

Comment:
Precisa validar o status da cobrança, senão for o status que é permitido deve emitir uma BE explicando